### PR TITLE
[WIP] First pass at new AZ scraper

### PR DESCRIPTION
**Pull request by showerst**
New Arizona Bill Scraper.

They seem to have gone backwards in some ways with their new site, I can't a real list of actions, or votes. This at least scrapes the bills, including 2017 prefiles.

A good project for someone new/TODO will be to check into action_utils.py -- I'm trying to map from the dates provided in the JSON to proper actions, but have a number of blanks for items I haven't researched yet.

### CODE REVIEW

**Comment by pelzhaus** [comment]
checkout the API endpoint on their api `api/BillStatusOverview/?billNumber={}&sessionId={}` to get actions and votes

**Comment by showerst** [comment]
@pelzhaus thanks I'll take a look at this over the holidays, leaving this example url for myself: 

https://apps.azleg.gov/api/BillStatusOverview/?billNumber=HB2001&sessionId=116

**Comment by pqnelson** [comment]
Legislators aren't working. The URL `www.azleg.gov/MemberRoster.asp?Session_ID=117&body=S` no longer works :(

It seems to have changed to `http://www.azleg.gov/MemberRoster/?body=S` ... but selecting the session does not seem immediately obvious (apparently it's handled server-side, whenever `.selectSession` changes -- c.f., [az.js](http://www.azleg.gov/azlegwp/wp-content/themes/azleg/js/az.js?ver=1.0.0)).

**Comment by jamesturk** [comment]
taking a look at this one now

File: openstates/az/action_utils.py [hunk 1]
```diff
@@ -1,212 +1,125 @@
...
-    'RETAINED': 'retained',
-
-    'RULE 8J PROPER': 'proper legislation and deemed not derogatory or insulting',
-
-    'S/C': 'subcommittee',
-    'S/C REPORTED': 'subcommittee reported',
-
-    'W/D': 'withdrawn',
-}
+action_map = {
```
Comment (showerst) [inline openstates/az/action_utils.py]: @jamesturk this chunk is my biggest concern, I tried to create a mapping between the various JSON keys the state provided, and name/actions. 

Not sure this approach is smart, so open to a refactor here if you've got better ideas.

**Pull request merged**